eidas_node/proxy_service/templates/eidas_node/proxy_service/identity_provider_response.html

```diff
@@ -0,0 +1,22 @@
+{% extends "eidas_node/proxy_service/base.html" %}
+{% load i18n staticfiles %}
+
+{% block title %}{% trans "Identity Provider Response" %}{% endblock %}
+{% block scripts %}
+    {{ block.super }}
+    <script type="text/javascript" src="{% static "eidas_node/proxy_service/formautosubmit.js" %}"></script>
+{% endblock %}
+
+{% block content %}
+    <h1>{% trans "Identity Provider Response" %}</h1>
+    {% if error %}
+        <p>{%  trans "An error occurred during processing Identity Provider response." %}</p>
```

Comment:
"An error occurred during processing ***of*** Identity Provider response."

Reply:
Fixed.